css/styles.css

```diff
@@ -1,51 +1,209 @@
-* {
-  font-family: Lato, sans-serif;
+.container {
+  font-family: "Proza Libre", sans-serif;
+}
+
+body {
+  background: #f0f0f0;
 }
 
 header {
-  background-color: #4b2373;
-  color: white;
-  padding: 15px;
-  font-family:
-    "Proza Libre",
-    sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 27px;
+  color: #fff;
+  background-color: #562b81;
+}
+
+.new-entry {
+  font-style: normal;
+  font-weight: 600;
+  font-size: 30px;
+  line-height: 41px;
+  padding-top: 16px;
+  padding-bottom: 9px;
 }
 
 img {
+  border-radius: 4px;
+  max-width: 100%;
+}
+
+.minor-heading {
+  font-style: normal;
+  font-weight: 600;
+  font-size: 16px;
+  line-height: 22px;
+  padding-bottom: 9px;
+  padding-top: 25px;
+  margin-top: 5px;
+}
+
+.small-box {
+  width: 100%;
+  height: 40px;
+  margin-bottom: 10px;
+}
+
+.large-box {
+  width: 100%;
+  height: 149px;
+}
+
+.inputs {
   width: 100%;
 }
 
-.form-actions {
+.button {
   display: flex;
   justify-content: flex-end;
 }
 
-label {
-  display: block;
+button {
+  font-family: "Proza Libre", sans-serif;
+  font-size: 16px;
+  color: #f0f0f0;
+  background: #562b81;
+  border-radius: 4px;
+  position: absolute;
```

Comment:
You shouldn't be positioning the button absolutely. It should be inside a flex container along with the "Entries" text